pymc3/sampling.py

```diff
@@ -1305,8 +1305,10 @@ def sample_prior_predictive(samples=500,
     model = modelcontext(model)
 
     if vars is None and var_names is None:
-        vars = set(model.named_vars.keys())
-        vars_ = model.named_vars
+        prior_pred_vars = model.observed_RVs
+        prior_vars = get_default_varnames(model.unobserved_RVs, include_transformed=True)
```

Comment:
This will exclude potentials (model.potentials)

Reply:
I did not know these existed. Should they be included in `prior_vars` or `prior_pred_vars`?

Reply:
Potentials are a way to add arbitrary terms to the computation of the logp. You can use them to add constraints to the model. I will add them to `prior_vars`